dashboard/test/ui/runner.rb

```diff
@@ -313,7 +320,9 @@ def output_synopsis(output_text)
 
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    if $options.log_dev
+      HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    end
```

Comment:
Yeah, I understand that. I think my main concern is that we already have a way of toggling this (CDO.hip_chat_logging) per environment. Our default behavior should be to respect that existing toggle, and only override it if we have an explicit override action (i.e. add the flag)